docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,312 @@
+# Monitor Kubernetes 
+
+Monitoring the state of the database is crucial to timely identify and react to performance issues. [Percona Monitoring and Management (PMM) solution enables you to do just that](monitoring.md).
+
+However, the database state also depends on the state of the Kubernetes cluster itself. Hence it’s important to have metrics that can depict the state of the Kubernetes cluster.
+
+This document describes how to set up monitoring of the Kubernetes cluster health. This setup has been tested with the [PMM server](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html#pmm-server) as the centralized data storage and the Victoria Metrics operator as the statistics collector, though the steps should also apply to other monitoring applications. 
+
+## Considerations
+
+1. In this setup we use [Victoria Metrics kubernetes monitoring stack](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack) Helm chart. When customizing the chart's values, consider the following:
+
+    * Since data is stored in the PMM server, disable VMSingle and VMCluster.
```

Comment:
```suggestion
    * Since we actually use PMM Server for monitoring and there is no need to actually store data in VictoriaMetrics, set `vmsngle.enabled` and `vmcuster.enabled` parameters of the Victoria Metrics Helm chart to `false`.
```